File: app/src/main/java/com/gs/buluo/store/bean/WalletAccount.java
package com.gs.buluo.store.bean;

import com.gs.buluo.store.bean.ResponseBody.IBaseResponse;

/**
 * Created by hjn on 2016/11/18.
 */
public class WalletAccount implements IBaseResponse {
    public enum STATUS {
        NORMAL("NORMAL"), LOCKED("LOCKED");
        String status;

        STATUS(String status) {
            this.status = status;
        }
    }

    public String id;
    public String balance;
    public STATUS state;
    public String lastTrading;
    public String password;
}

File: app/src/main/java/com/gs/buluo/store/bean/WalletAccount.kt
package com.gs.buluo.store.bean

import com.gs.buluo.store.bean.ResponseBody.IBaseResponse

/**
 * Created by hjn on 2016/11/18.
 */
class WalletAccount(var id: String, var balance: Float, var withdrawCharge: Float, var state: STATUS, var lastTrading: String, var password: String
,var accountType: AccountType) : IBaseResponse {
    enum class STATUS private constructor(internal var status: String) {
        NORMAL("NORMAL"), LOCKED("LOCKED")
    }

    enum class AccountType private constructor(){
        CARD,PROTOCOL
    }
}
